packages/vite-plugin-checker/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import chalk from 'chalk'
 import { spawn } from 'child_process'
-import pick from 'lodash.pick'
+import { pick } from 'lodash'
```

Comment:
better use direct import, because lodash cannot properly tree-shaked
```suggestion
import pick from 'lodash/pick'
```

Reply:
I have read that it might not be so relevant depending on the bundler, but I think you're right though, and this should do no harm :)